src/libraries/System.Security.Cryptography.X509Certificates/tests/RevocationTests/DynamicRevocationTests.cs

```diff
@@ -364,7 +364,7 @@ public static void RevokeEverything(PkiOptions pkiOptions)
                 });
         }
 
-        [Theory]
+        [ConditionalTheory(typeof(SignatureSupport), nameof(SignatureSupport.SupportsRsaSha1Signatures))]
```

Comment:
This test doesn't appear to have anything to do with SHA-1, it should not be disabled